test/e2e/support/utils.js

```diff
@@ -235,19 +244,21 @@ export async function insertBlock( searchTerm, panelName = null ) {
  * Performs a key press with modifier (Shift, Control, Meta, Mod), where "Mod"
  * is normalized to platform-specific modifier (Meta in MacOS, else Control).
  *
- * @param {string} modifier Modifier key.
- * @param {string} key      Key to press while modifier held.
- *
- * @return {Promise} Promise resolving when key combination pressed.
+ * @param {string|Array} modifiers Modifier key or array of modifier keys.
+ * @param {string} key      	   Key to press while modifier held.
  */
-export async function pressWithModifier( modifier, key ) {
-	if ( modifier.toLowerCase() === 'mod' ) {
-		modifier = MOD_KEY;
-	}
+export async function pressWithModifier( modifiers, key ) {
+	const modifierKeys = castArray( modifiers );
+
+	await Promise.all(
+		modifierKeys.map( async ( modifier ) => page.keyboard.down( modifier ) )
```

Comment:
Minor: There's no need for this to be an `async` function, since the `await` keyword is not used within the function's body.